Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -142,6 +200,11 @@ private void OnUnstickSuccessful(UnstickSuccessfulEvent ev)
             return;
 
         UnstickFromEntity(ev.Uid, ev.User, component);
+
+        if (EntityManager.HasComponent<HasEntityStuckOnComponent>(ev.Target))
+            EntityManager.RemoveComponent<HasEntityStuckOnComponent>(ev.Target);
```

Comment:
```suggestion
        EntityManager.RemoveComponent<HasEntityStuckOnComponent>(ev.Target);
```

removecomp alraedy checks for this now